nux/components/dot-tip/README.md

```diff
@@ -0,0 +1,31 @@
+DotTip
+========
+
+`DotTip` is a React component that renders a single _tip_ on the screen. The tip will point to the React element that `DotTip` is nested within. Each tip is uniquely identified by a string passed to `id`.
+
+## Usage
+
+```jsx
+<button onClick={ ... }>
+	Add to Cart
+	<DotTip id="acme/add-to-cart">
+		Click here to add the product to your shopping cart.
+	</DotTip>
+</button>
+}
+```
+
+## Props
+
+The component accepts the following props:
+
+### id
+
+An identifier that uniquely identifies the tip.
```

Comment:
Should we want to establish/document naming conventions?

Reply:
👌 524f5b8db